lib/index.js

```diff
@@ -98,17 +119,17 @@ function StickyInlines(opts) {
    * @return {Change}
    */
 
-  function onSelect(event, change) {
-    const selection = findRange(window.getSelection(), change.value);
-    const selectionIsAtEndOfInline = change.value.focusInline &&
+  function onSelect(event, change, next) {
+    const selection = findRange(window.getSelection(), change.editor);
```

Comment:
I'm pretty sure the editor does not have an editor property. Should this be editor.value?